livekit-agents/pyproject.toml

```diff
@@ -27,6 +27,7 @@ classifiers = [
 ]
 dependencies = [
     "click~=8.1",
+    "certifi>=2025.6.15",
```

Comment:
Is it being used by default tho?

Reply:
yes. we need SSL root certs in order to connect to anything https. `requests`, `httpx`, `httpcore` all depends on this with a versionless dependency.

the problem is the default version on some systems are too old that they cannot connect to cloud. it's more of an issue on Mac dev machines, since some OS bundle an really old version of python